src/main/java/mil/dds/anet/beans/ApprovalStep.java

```diff
@@ -31,6 +31,9 @@
   @GraphQLQuery
   @GraphQLInputField
   String name;
+  @GraphQLQuery
+  @GraphQLInputField
+  private boolean restrictedApproval;
```

Comment:
This makes it a required fiend for GraphQL, which does not work when working with Approval Steps of anything but tasks. The following workaround helps:
```
diff --git a/src/main/java/mil/dds/anet/beans/ApprovalStep.java b/src/main/java/mil/dds/anet/beans/ApprovalStep.java
index 4c3342e5e..bcdedd0ea 100644
--- a/src/main/java/mil/dds/anet/beans/ApprovalStep.java
+++ b/src/main/java/mil/dds/anet/beans/ApprovalStep.java
@@ -33,7 +33,7 @@ public class ApprovalStep extends AbstractAnetBean {
   String name;
   @GraphQLQuery
   @GraphQLInputField
-  private boolean restrictedApproval;
+  private Boolean restrictedApproval;
 
   @GraphQLQuery(name = "approvers")
   public CompletableFuture<List<Position>> loadApprovers(
@@ -90,7 +90,7 @@ public class ApprovalStep extends AbstractAnetBean {
   }
 
   public boolean isRestrictedApproval() {
-    return restrictedApproval;
+    return Boolean.TRUE.equals(restrictedApproval);
   }
 
   public void setRestrictedApproval(boolean restrictedApproval) {
```